x-pack/metricbeat/module/istio/citadel/_meta/fields.yml

```diff
@@ -0,0 +1,59 @@
+- name: citadel
+  type: group
+  description: >
+    Contains statistics related to the Istio Citadel service
+  release: beta
+  fields:
+    - name: grpc.method
+      type: keyword
+      description: >
+        The grpc method
+    - name: grpc.service
+      type: keyword
+      description: >
+        The grpc service
+    - name: grpc.type
+      type: keyword
+      description: >
+        The type of the respective grpc service
+
+    - name: secret_controller_svc_acc_created_cert_count
```

Comment:
This must be `secret_controller_svc_acc_created_cert.count` to follow naming conventions

Reply:
Thanks for mentioning. Will change it.